_layouts/security.html

```diff
@@ -0,0 +1,54 @@
+        <h1>Reporting of CVEs and Security Issues</h1>
+        <h2 class="page-subtitle">The Wild<strong>Fly</strong> community and our sponsor, Red Hat, take security bugs very
+            seriously
+        </h2>
+        <p>We aim to take immediate action to address serious security-related problems that involve our projects.
+            Note that we will only fix such issues in the most recent major release of Wild<strong>Fly</strong>.</p>
+    </div>
+    <div class="grid__item width-12-12 sec-title-red">
+        <h2>Reporting of Security Issues</h2>
+        <p>When reporting a security vulnerability it is important to not accidentally broadcast to the world that the
+            issue exists, as this makes it easier for people to exploit it. The software industry uses the term
+            <a href="https://www.redhat.com/en/blog/security-embargoes-red-hat">embargo</a> to describe the time a security
+            issue is known internally until it public knowledge.
+        </p>
+        <p>
+            Our preferred way of reporting security issues in Wild<strong>Fly</strong> and its related projects are listed below.
+        </p>
+    </div>
+    <div class="grid__item width-12-12 sec-title-teal">
+        <h2>Email the mailing list</h2>
+        <p>The list at <a href="mailto:XXX@redhat.com">XXX@redhat.com</a> is the preferred mechanism for outside users
+            to report security issues. This A member of the Wild<strong>Fly</strong> team will open the required issues.</p>
+    </div>
+    <div class="grid__item width-12-12 sec-title-yellow">
+        <h2>Open a Jira issue</h2>
+        <p>If you are an experienced contributor, and choose to open a Jira issue in the correct project at
+            <a href="https://issues.redhat.com">https://issues.redhat.com</a>, make sure to set the
+            <strong>Security Level</strong> field to <strong>Embargoed Security Issue</strong>. This keeps it private.
+        </p>
+        <p>
+            Note that if you do open a Jira issue with the Security Level set to Embargoed Security Issue, you will not be able
+            to view the issue due to our security policy.
+        </p>
+    </div>
+    <div class="grid__item width-12-12 sec-title-red">
+        <h2>Other considerations</h2>
+        <p>
+            It might be tempting to attempt to fix the security issue, especially if it seems like something easy.
+            If you do do this, please make sure that you <strong>do not open any public pull requests</strong> since
+            that risks breaking the embargo.
+        </p>
+        <p>
+            Instead, make a private fork of the repository, and send us a message via <a href="mailto:XXX@redhat.com">XXX@redhat.com</a>.
+            A member of the team will be in touch. Once that happens, you will need to add them as a contributor to your private
+            fork so that they can see your changes.
+        </p>
+    </div>
+</div>
```

Comment:
I like what https://www.keycloak.org/security.html does about this, if we can work that out. It's simplest -- send us your info, we then get back to you on how to work the issue privately.

Reply:
@bstansberry updated